apps/modernization-ui/src/pages/patient/profile/administrative/AdministrativeTable.tsx

```diff
@@ -44,8 +42,8 @@ type Props = {
 export const AdministrativeTable = ({ patient }: Props) => {
     const { showAlert } = useAlert();
     const [tableHead, setTableHead] = useState<{ name: string; sortable: boolean; sort?: string }[]>([
-        { name: 'As of', sortable: true, sort: 'all' },
-        { name: 'General comment', sortable: true, sort: 'all' },
+        { name: 'As of', sortable: true },
```

Comment:
the tableHeader can be used as a variable and you can import it from the interface. you can use this example Im pasting from the **IdentificationsTable.tsx** 

const headers = [
    { name: Column.AsOf, sortable: true },
    { name: Column.Type, sortable: true },
    { name: Column.Authority, sortable: true },
    { name: Column.Value, sortable: true },
    { name: Column.Actions, sortable: false }
];